tests/auto/ci_auto.py

```diff
@@ -54,18 +54,19 @@ def __init__(self):
 
 
 def set_action_from_label(machine, actions, label):
-    ''' Match the label that initiates a job with an action in the dict'''
-    # <machine>-<compiler>-<test> i.e. hera-gnu-RT
+    ''' Match the label that initiates a job with an action in the dict
+        Labels have a ci- prefix'''
+    # ci-<machine>-<compiler>-<test> i.e. ci-hera-intel-build
     logger = logging.getLogger('MATCH_LABEL_WITH_ACTIONS')
-    logger.info('Setting action from Label')
+    logger.info('Setting action from Label {label}')
     split_label = label.name.split('-')
     # Make sure it has three parts
-    if len(split_label) != 3:
+    if len(split_label) != 4:
         return False, False
-    # Break the parts into their variables
-    label_machine = split_label[0]
-    label_compiler = split_label[1]
-    label_action = split_label[2]
+    # Break the parts into their variables, ignoring prefix
+    label_machine = split_label[1]
+    label_compiler = split_label[2]
+    label_action = split_label[3]
```

Comment:
Just a syntax suggestion to reduce mental overhead of tracking list positions with numbers.

```
_, label_machine, label_compiler, label_action = split_label
```